src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Colors {
+    BLACK,
+    WHITE,
+    RED,
+    BLUE,
+    GREEN,
+    MAGENTA,
+    YELLOW,
+    GERY
```

Comment:
There seems to be a typo in the enum value 'GERY'. It should be spelled as 'GRAY' to correctly represent the color.